src/yamlStream.ts

```diff
@@ -47,27 +47,40 @@ const levelName = (level: number): string | undefined => {
   }
 };
 
+function indent(str: string, spaceCount: number = 2): string {
+  if (str.length === 0)
+    return str;
+  const indent = ' '.repeat(spaceCount);
+  return str.replace(/^(.+)$/mg, `${indent}$1`);
+}
+
+
 const formatError = (basePath: RegExp, err?: BunyanRecord['err']): string => {
   if (!err)
     return '';
   let stack = err.stack;
 
   if (err.ice_name) {
-    const {ice_name, ice_cause} = err;
-    const header = `Error: ${ice_name}: ${ice_cause}`;
+    const {ice_name, ice_cause, message} = err;
+    const header = `Error: ${ice_name}: ${ice_cause || message || ''}`;
     stack = header + '\n' + stack.replace(/.*?\n/, '');
   }
 
-  return stack.replace(basePath, '') + '\n';
+  const stackLines = stack.split('\n');
+  const errorContext = {
+    context: omit(err, ['stack', 'message', 'ice_name', 'ice_cause']),
+  };
+  const stackTrace = stackLines.slice(1).map(
+    line => line.replace(basePath, '')
+  );
```

Comment:
Message could contain any possible string so we should not modify message anyway